src/docfx/Errors.cs

```diff
@@ -120,8 +120,8 @@ public static Error GitShadowClone(string repoPath)
         public static Error GitNotFound()
             => new Error(ErrorLevel.Error, "git-not-found", $"Cannot find git, install git https://git-scm.com/");
 
-        public static Error BookmarkNotFound(Document relativeTo, Document reference, string bookmark)
-            => new Error(ErrorLevel.Warning, "bookmark-not-found", $"Cannot find bookmark '#{bookmark}' in '{reference}'", relativeTo.ToString());
+        public static Error BookmarkNotFound(Document relativeTo, Document reference, string bookmark, string suggestedBookmark = null)
+            => new Error(ErrorLevel.Warning, "bookmark-not-found", $"Cannot find bookmark '#{bookmark}' in '{reference}'{(!string.IsNullOrEmpty(suggestedBookmark) ? $", did you mean '{suggestedBookmark}'" : null)}", relativeTo.ToString());
```

Comment:
nit: consider having a `Hint` helper here:

`$"Cannot find a bookmark in '{reference}'{Hint(bookmark, candidates)}."`